bundles/org.openhab.binding.warmup/src/main/java/org/openhab/binding/warmup/internal/WarmupBindingConstants.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
```

Comment:
The copyright year needs to be updated. You can use `mvn license:format -pl :org.openhab.binding.warmup` in the root. You need to rebase you binding to the current main to update (`git pull --rebase`) the configuration containing the new year before running this and than `git push --force-with-lease` update your remote branch,